### PR TITLE
Force update on timelinepanel when event decrypted

### PR DESCRIPTION
**Pull request by dbkr**
Partially fixes https://github.com/vector-im/riot-web/issues/4865
(the part about messages not appearing in the timeline until you
move the cursor over the window).

Requires https://github.com/matrix-org/matrix-js-sdk/pull/529

### CODE REVIEW

File: src/components/structures/TimelinePanel.js [hunk 1]
```diff
@@ -503,6 +505,16 @@ var TimelinePanel = React.createClass({
...
     },
 
+    onEventDecrypted: function(ev) {
+        // Need to update as we don't display event tiles for events that
+        // haven't yet been decrypted. The event will have just been updated
+        // in place so we just need to re-render.
+        // TODO: We should restrict this to only events in our timeline,
+        // but possibly the event tile itself should just update when this
+        // happens to save us re-rendering the whole timeline.
+        this.forceUpdate();
```
Comment (richvdh) [inline src/components/structures/TimelinePanel.js]: hrm; surely we should at least check the room id of the event?

Comment (dbkr) [inline src/components/structures/TimelinePanel.js]: Yeah, good point. Done.

**Comment by lukebarnard1** [comment]
I wonder if it's worth debouncing the forced update?

**Comment by dbkr** [comment]
@lukebarnard1 possibly, but we don't do it onRoomTimeline so if that hasn't been causing issues, I'm going to say it's probably better to leave it out for now.

**Pull request merged**